File: fabric/src/main/java/net/impleri/itemskills/client/fabric/ItemSkillsClientFabric.java
package net.impleri.itemskills.client.fabric;

import net.fabricmc.api.ClientModInitializer;
import net.impleri.itemskills.client.ItemSkillsClient;

public class ItemSkillsClientFabric implements ClientModInitializer {
    @Override
    public void onInitializeClient() {
        ItemSkillsClient.init();
    }
}

File: fabric/src/main/kotlin/net/impleri/itemskills/client/fabric/ItemSkillsClientFabric.kt
package net.impleri.itemskills.client.fabric

import net.fabricmc.api.ClientModInitializer
import net.impleri.itemskills.client.ItemSkillsClient

class ItemSkillsClientFabric : ClientModInitializer {
  override fun onInitializeClient() {
    ItemSkillsClient.init()
  }
}
